### PR TITLE
Add documentation for running the tests to the README.

### PR DESCRIPTION
**Pull request by rwjblue**
This adds the current test command to the README while we work on a better solution over in https://github.com/notion-cli/notion/issues/168.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,14 @@ We use [Discord](https://discordapp.com/) for community discussion. You can use
 
 Notion is intended to compile with all versions of Rust newer than 1.20.
 
+## Tests
+
+To run the tests in this repo, run the following:
+
+```
+cargo test --features mock-network
```
Comment (mikrostew) [inline README.md]: This should be `cargo test --all --features mock-network`, to include all the unit tests from the sub-crates in this project

Comment (locks) [inline README.md]:
```suggestion
cargo test --all --features mock-network
```

Comment (locks) [reply README.md]: :angel: trying out the new feature.

Comment (rwjblue) [reply README.md]: Sorry about that, thank you for pointing out the correct command!

Comment (rwjblue) [reply README.md]: Whoops, they didn't send me a notification for the suggested fix 😕 and I clobbered it...

Comment (mikrostew) [reply README.md]: Oh that's an interesting new thing!

**Pull request merged**